README.md

```diff
@@ -174,7 +174,7 @@ React native screens library is licensed under [The MIT License](LICENSE).
 
 ## Credits
 
-This project has been build and is maintained thanks to the support from [Shopify](https://shopify.com), [Expo.io](https://expo.io) and [Software Mansion](https://swmansion.com)
+This project has been build and is maintained thanks to the support from [Shopify](https://shopify.com), [Expo.io](https://expo.io), and [Software Mansion](https://swmansion.com).
```

Comment:
Thanks for your contribution and making this library better :tada: Are you sure there should be a coma before `and` in this case?

Reply:
I'm pretty sure - https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/
Although, in this particular case I think it can be skipped. I think it's generally better to remember about always adding it than thinking if it can be skipped or not.

Reply:
Ok I get it all now, thanks for the valuable insight on the topic.

Reply:
Although, I think the description can be a bit misleading, since the PR adds more changes than it is written there.

Reply:
Updated the description :)